kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/BigQueryWriter.java

```diff
@@ -123,10 +133,24 @@ public void writeRows(
         if (err.code() == INTERNAL_SERVICE_ERROR || err.code() == SERVICE_UNAVAILABLE) {
           // backend error: https://cloud.google.com/bigquery/troubleshooting-errors
           retryCount++;
+        } else if (err.code() == FORBIDDEN
+                   && err.error() != null
+                   && QUOTA_EXCEEDED_REASON.equals(err.error().reason())) {
+          // quota exceeded error
```

Comment:
Would be good to have at least a debug log here, so we know when we're hitting quotas.

Also would be good to have a metric to count retries.